src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -2,33 +2,39 @@
 
 public class defaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+    private static final int NUMBER = 12;
+    private static final int EMPTY_VALUE = -1;
+    private int[] numbers = new int[NUMBER];
 
-    public int total = -1;
+    public int getTotal() {
+        return total;
+    }
+
+    private int total = EMPTY_VALUE;
 
     public void countIn(int in) {
         if (!isFull())
-            NUMBERS[++total] = in;
+            numbers[++total] = in;
     }
 
-        public boolean callCheck() {
-            return total == -1;
-        }
-        
-            public boolean isFull() {
-                return total == 11;
-            }
-        
-                protected int peekaboo() {
-                    if (callCheck())
-                        return -1;
-                    return NUMBERS[total];
-                }
-            
-                    public int countOut() {
-                        if (callCheck())
-                            return -1;
-                        return NUMBERS[total--];
-                    }
+    public boolean callCheck() {
+        return total == EMPTY_VALUE;
```

Comment:
This constant should be different (separate) than the one in peekaboo and countOut methods.